README.md

```diff
@@ -166,58 +166,6 @@ NNTrainer provides
 Currently, we provide [C APIs](https://github.com/nnstreamer/nntrainer/blob/master/api/capi/include/nntrainer.h) for Tizen. [C++ APIs](https://github.com/nnstreamer/nntrainer/blob/master/api/ccapi/include) are also provided for other platform. Java & C# APIs will be provided soon.
 
 
-### Examples for NNTrainer
```

Comment:
How about link to Application/README.md?

Reply:
Yes. I Linked ```nntrainer/Application/README.md``` page to "Examples for NNTrainer"